src/Workleap.DomainEventPropagation.Topics/Workleap.DomainEventPropagation.Topics.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+      <!-- Temporarily disable public API analyzers until API has been stabilized -->
+    <NoWarn>RS0016,RS0037,$(NoWarn)</NoWarn>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>../Workleap.DomainEventPropagation.snk</AssemblyOriginatorKeyFile>
+    <RootNamespace>Workleap.DomainEventPropagation</RootNamespace>
+  </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\Workleap.DomainEventPropagation.Abstractions\Workleap.DomainEventPropagation.Abstractions.csproj" PackageVersion="[1.1.1,)"  />
+    </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="3.3.4">
```

Comment:
You will need to add the `PublicApi.Shipped.txt` and `PublicApi.Unshipped.txt` files manually 🚀

Reply:
ah indeed could add them already